### ISSUE TITLE
docker-app helm fails helm lint.

### ISSUE THREAD

**Issue by fibonacci1729** [docker/app#257]
**Description**
`docker-app helm` output isn't compliant with helm lint. The name of the chart of directory is expected to be the name of the chart.

**Steps to reproduce the issue:**
1. docker-app helm
2. helm lint

**Comment by garethr** [comment]
@fibonacci1729 thanks for the heads up. We've started to fix this in https://github.com/docker/app/pull/259

**Comment by ijc** [comment]
I think this was "fixed" in #437 which removed helm support:
```console
$ ./bin/docker-app helm
Error: unknown command "helm" for "docker-app"
Run 'docker-app --help' for usage.
```

Hopefully any future CNAB based Heml support which gets introduced won't replicate this issue, in the meantime I'm going to close. Thanks for your report @fibonacci1729.
